client/components/account-status/account-tools/strings.tsx

```diff
@@ -4,12 +4,22 @@
  */
 import { __ } from '@wordpress/i18n';
 
+/**
+ * Internal dependencies
+ */
+import { isInDevMode } from 'utils';
+
 export default {
 	title: __( 'Account Tools', 'woocommerce-payments' ),
-	description: __(
-		'Payments and deposits are disabled until account setup is completed. If you are experiencing problems completing account setup, or need to change the email/country associated with your account, you can reset your account and start from the beginning.',
-		'woocommerce-payments'
-	),
+	description: isInDevMode()
+		? __(
+				'You are in dev mode. If you are experiencing problems completing account setup, or need to change the email/country associated with your account, you can reset your account and start from the beginning.',
```

Comment:
We use `sandbox mode` instead of `dev mode` in our copy to prevent leaking out internal idioms. 

I would change this copy to:
"Your account is in sandbox mode. If you are experiencing problems completing account setup, or wish to test with a different email/country associated with your account, you can reset your account and start from the beginning."